insights/specs/datasources/team.py

```diff
@@ -0,0 +1,47 @@
+"""
```

Comment:
@TZ3070 - I believe your method works as expected, however, we generally implement the dependencies via `component` where there are existing parsers can be leveraged.

Please check the following examples:
https://github.com/RedHatInsights/insights-core/blob/b12686290b8962b2f504efebef3ebb86cf3d733f/insights/specs/default.py#L75

https://github.com/RedHatInsights/insights-core/blob/b12686290b8962b2f504efebef3ebb86cf3d733f/insights/specs/default.py#L109

would you please change this datasource to `compoent`?